score/mw/log/detail/file_logging/dlt_message_builder.cpp

```diff
@@ -17,9 +17,9 @@
 #include "score/mw/log/detail/file_logging/svp_time.h"
 #include "score/mw/log/log_level.h"
 
-#include "static_reflection_with_serialization/serialization/for_logging.h"
-#include "static_reflection_with_serialization/visitor/visit.h"
-#include "static_reflection_with_serialization/visitor/visit_as_struct.h"
+#include "serialization/for_logging.h"
+#include "visitor/visit.h"
```

Comment:
Some use `#include <visitor/visit.h>`, some are like this ^. I would make them consistent.